prep-javascript-functions/main.js

```diff
@@ -0,0 +1,48 @@
+function addTwoNumbers(num1, num2) {
+  return num1 + num2;
+}
+
+var sum = addTwoNumbers(4, 4);
+console.log('sum', sum);
+
+function convertHoursToMinutes(hours) {
+  console.log('hours: ' + hours + ', minutes: ' + hours * 60);
+}
+
+convertHoursToMinutes(2);
+
+function getGreeting(name) {
+  console.log('Hello ' + name + '!');
+}
+
+getGreeting('World');
+
+function addAndMultiplyBy5(num1, num2) {
+  console.log('(', num1 + ' + ' + num2, ')' + ' % ', ' 5 ', ' = ', ((num1 + num2) * 5));
+}
+
+addAndMultiplyBy5(10, 5);
+
+function subtractTwoNumbers(num1, num2) {
+  console.log(num1 + ' - ' + num2 + ' = ' + (num1 - num2));
+}
+
+subtractTwoNumbers(22, 7);
+
+function getCircleCircumference(radius) {
+  console.log('radius of ' + radius + ' = ' + (2 * Math.PI) * radius);
+}
+
+getCircleCircumference(5);
+
+function getFullName(firstName, lastName) {
+  console.log(firstName + ' ' + lastName);
+}
+
+getFullName('Lance', 'Cruzado');
+
+function cube(number) {
+  console.log('cube of ' + number + ' = ' + (number * number * number));
+}
```

Comment:
Hi Lance, your functions should `return` the result instead of console logging it, like the example provided in the instructions.


![](https://s3.us-west-1.amazonaws.com/lfz-lms/exercise-instructions-assets/Learning-Fuze/curriculum/3025d071531539bf167a22ea4e412e2d99140460/assets/how-to-log-result.gif)
For example this function should not concatenate any strings, it should just do the calculation and return the result. Then outside of the function you log the return value of calling the function
```js
function addAndMultiplyBy5(num1, num2) {
  console.log('(', num1 + ' + ' + num2, ')' + ' % ', ' 5 ', ' = ', ((num1 + num2) * 5));
}
```

Reply:
Ok! thank you! I made the changes and pushed it